src/components/auction/OrdersTable/index.tsx

```diff
@@ -144,6 +144,11 @@ const OrderTable: React.FC<OrderTableProps> = (props) => {
   )
 
   const pendingText = `Cancelling Order`
+  const orderStatusText = {
+    [OrderStatus.PLACED]: 'Placed',
+    [OrderStatus.PENDING]: 'Pending',
+    [OrderStatus.PENDING_CANCELLATION]: 'Pending-Cancelation',
```

Comment:
Now, seeing it, probably Pending-Canelation is too long. We should probably just call it `Cancelling`.

Sorry for the bad proposal :) in the first place

Reply:
Thanks for looking at it, no problem.